node-java/Dockerfile

```diff
@@ -4,6 +4,7 @@ RUN curl -sL https://deb.nodesource.com/setup_8.x | bash
 
 RUN apt-get -y update  && \
     apt-get -y upgrade && \
-    apt-get -y install nodejs
+    apt-get -y install nodejs=8.* \
+                       make
```

Comment:
Is it necessary to have that much whitespace before calling make?

Reply:
No, just trying to keep it inline with above.